apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/page-patterns-plugin.tsx

```diff
@@ -53,8 +55,8 @@ export function PagePatternsPlugin( props: PagePatternsPluginProps ) {
 
 	const { getMeta, postContentBlock } = useSelect( ( select ) => {
 		const getMeta = () =>
-			( select( 'core/editor' ) as CoreEditorPlaceholder ).getEditedPostAttribute( 'meta' );
-		const currentBlocks = ( select( 'core/editor' ) as CoreEditorPlaceholder ).getBlocks();
+			( select( editorStore ) as CoreEditorPlaceholder ).getEditedPostAttribute( 'meta' );
+		const currentBlocks = select( blockEditorStore ).getBlocks();
```

Comment:
Did you mean `editorStore`?

```suggestion
		const currentBlocks = select( editorStore ).getBlocks();
```